### PR TITLE
Push minor branch into major

### PR DESCRIPTION
**Pull request by driusan**
This attempts to push the minor branch into major. The following files had conflicts and should be tested extra carefully:

	both modified:   composer.json
	both modified:   modules/api/php/endpoints/project/candidates.class.inc
	both modified:   modules/api/php/endpoints/project/instrument/instrument.class.inc
	both modified:   modules/api/php/endpoints/project/instruments.class.inc
	both modified:   modules/api/php/endpoints/project/project.class.inc
	both modified:   modules/api/php/endpoints/project/visits.class.inc
	both modified:   modules/api/php/endpoints/projects.class.inc
	both modified:   modules/bvl_feedback/ajax/react_get_bvl_threads.php
	both modified:   modules/dashboard/test/DashboardTest.php
	both modified:   modules/imaging_browser/php/imaging_browser.class.inc
	both modified:   modules/imaging_browser/test/imaging_browserTest.php
	both modified:   modules/login/php/passwordexpiry.class.inc
	both modified:   modules/media/php/media.class.inc
	both modified:   modules/mri_violations/test/mri_violationsTest.php
	both modified:   modules/statistics/php/stats_behavioural.class.inc
	both modified:   modules/statistics/php/stats_demographic.class.inc
	both modified:   modules/statistics/php/stats_mri.class.inc
	both modified:   modules/survey_accounts/test/survey_accountsTest.php
	both modified:   modules/timepoint_list/test/timepoint_listTest.php
	both modified:   modules/user_accounts/php/edit_user.class.inc
	both modified:   modules/user_accounts/php/my_preferences.class.inc
	deleted by us:   package-lock.json
	both modified:   php/libraries/Candidate.class.inc
	both modified:   php/libraries/File_Upload.class.inc
	both modified:   php/libraries/NDB_Page.class.inc
	both modified:   php/libraries/SinglePointLogin.class.inc
	both modified:   php/libraries/TimePoint.class.inc
	both modified:   php/libraries/Utility.class.inc
	deleted by them: src/Api/Endpoints/Project/Images.php
	both modified:   test/run-php-linter.sh

### CODE REVIEW

**Comment by johnsaigle** [comment]
This file list looks familiar... I think it's mainly PHPCS changes if I'm not mistaken.

But yes, care should be taken.

File: modules/survey_accounts/test/survey_accountsTest.php [hunk 1]
```diff
@@ -151,7 +152,7 @@ public function setUp()
                 'ID'                    => '111111',
                 'CandID'                => '999888',
                 'CenterID'              => '55',
-                'RegistrationProjectID' => '7777',
+                'ProjectID'             => '7777',
```
Comment (johnsaigle) [inline modules/survey_accounts/test/survey_accountsTest.php]: Isn't `RegistrationProjectID` the newer change?

Comment (ridz1208) [reply modules/survey_accounts/test/survey_accountsTest.php]: this is in the session table

**Comment by kongtiaowang** [comment]
@driusan I am debugging the Error in media.

**Pull request merged**